mr.user.js

```diff
@@ -82,78 +102,80 @@ const setDiscussionInfo = async (mergeRequest) => {
 	return mergeRequest;
 };
 
-const getTime = (element) => Date.parse(element.find('time').attr('datetime'));
+const getTime = (element) => Date.parse(element.querySelector('time').getAttribute('datetime'));
 
-const hasToTestLabel = (element) => element
-	.find('.gl-label-text')
-	.filter((index, elem) => toTestLabels.includes(elem.innerText.trim()))
-	.length > 0;
+/**
+ * @param {Element} element
+ * @return {boolean}
+ */
+const hasToTestLabel = (element) => {
+	return Array.from(element.querySelectorAll('.gl-label-text'))
+		.filter(elem => toTestLabels.includes(elem.textContent.trim()))
+		.length > 0;
+};
 
-const hasTestDoneLabel = (element) => element
-	.find('.badge')
-	.filter((index, elem) => testDoneLabels.includes(elem.innerText.trim()))
-	.length > 0;
+/**
+ * @param {Element} element
+ * @return {boolean}
+ */
+const hasTestDoneLabel = (element) => {
+	return Array.from(element.querySelectorAll('.badge'))
+		.filter(elem => testDoneLabels.includes(elem.textContent.trim()))
+		.length > 0;
+};
 
-const hasMergeConflictsMark = (element) => element.find('.issuable-pipeline-broken').length > 0;
-const getLikes = (element) => parseInt(element.find('.issuable-upvotes').text(), 0) || 0;
-const isWip = (mergeRow) => mergeRow.find('.merge-request-title-text a').text().includes('WIP');
-const isMergeRequestPage = () => $('.merge-request-tabs').length > 0;
-const getIssuableReference = (mergeRow) => mergeRow.find('.issuable-reference').text().replace(/\D/g, '');
-const getMergeRequestTitleTextElement = (mergeRow) => mergeRow.find('.merge-request-title-text');
-const getMergeDescription = (mergeRow) => mergeRow.find('.issuable-info');
+const hasMergeConflictsMark = (mr) => mr.querySelectorAll('.issuable-pipeline-broken').length > 0;
+const isWip = mr => mr.querySelector('.merge-request-title-text a').textContent.includes('WIP');
```

Comment:
```suggestion
const isWip = (mr) => mr.querySelector('.merge-request-title-text a').textContent.includes('WIP');
```
Если не ошибаюсь, то eslint на такое ругается.